index.js

```diff
@@ -2,12 +2,17 @@
 const stripAnsi = require('strip-ansi');
 const charRegex = require('char-regex');
 
-const stringLength = string => {
+const stringLength = (string, options = {}) => {
```

Comment:
```suggestion
const stringLength = (string, {ignoreAnsiEscapeCodes = false} = {}) => {
```

Reply:
Shouldn't this be something like `countAnsiEscapeCodes = true`?

Reply:
Fair point, not sure why I went with a negative phrasing. Will change.